src/main/java/Logger.java

```diff
@@ -75,12 +77,20 @@ public static void print(String msg) {
 
     /**
      * Prints an error string in the console if allowed.
+     * Returns true if the user requested to exit the application
      *
      * @param msg given error message.
      */
     public static void printError(String msg) {
         if (getInstance().mayPrint()) System.err.println(msg);
         getInstance().writeBuffer(msg);
+        
+        if(Properties.showErrorDialogue()) {
```

Comment:
In a vacuum I would tend to agree, but in this specific case I think it would be more confusing having to use a method of the Gui class to log errors and a method from the logger directly to log regular Messages. I'd rather always have the Logger methods be called.